VMEncryption/main/patch/centosPatching.py

```diff
@@ -74,39 +74,6 @@ def __init__(self, logger, distro_info):
             self.umount_path = '/usr/bin/umount'
 
     def install_extras(self):
-        epel_packages_installed = False
-        attempt = 0
-
-        while not epel_packages_installed:
-            attempt += 1
-            self.logger.log("Attempt #{0} to locate EPEL packages".format(attempt))
-            if self.distro_info[1].startswith("6."):
-                if self.command_executor.Execute("rpm -q ntfs-3g python-pip"):
-                    epel_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm"
-
-                    if self.command_executor.Execute("rpm -q epel-release"):
-                        self.command_executor.Execute(epel_cmd)
-
-                    self.command_executor.Execute("yum install -y ntfs-3g python-pip")
-
-                    if not self.command_executor.Execute("rpm -q ntfs-3g python-pip"):
-                        epel_packages_installed = True
-                else:
-                    epel_packages_installed = True
-            else:
-                if self.command_executor.Execute("rpm -q ntfs-3g python2-pip"):
-                    epel_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
-
-                    if self.command_executor.Execute("rpm -q epel-release"):
-                        self.command_executor.Execute(epel_cmd)
-
-                    self.command_executor.Execute("yum install -y ntfs-3g python2-pip")
-
-                    if not self.command_executor.Execute("rpm -q ntfs-3g python2-pip"):
-                        epel_packages_installed = True
-                else:
-                    epel_packages_installed = True
-
         packages = ['cryptsetup',
```

Comment:
do we understand all the packages and there need here?

Reply:
- ntfs-3g package is not required since the key volume is always vfat 
- python-pip package was used SUSE patching to install JWT in the ADAL scenario no longer required 
- epel was added because it includes dependencies required by python-pip which is no longer required 
- some code comments explaining the inclusion of other packages still being included would be helpful, I will work on including that.

Reply:
- lsscsi provides scsi device identifier information 
- psmisc provides utilities like fuser to identify what processes are using files 
- cryptsetup-reencrypt provides dm-crypt utilities 
- lvm2 provides lvm support
- uuid Simple, fast generation of RFC4122 UUIDS
- at for command scheduling
- patch provides utility for patching (may now be candidate for removal post Dracut design change)
- procps-ng provides utilities for browsing procfs and entries in process table
- util-linux standard package including utilities like blkid and lsblk 
- python-six wraps differences between python 2 and 3 
- pyparted includes python bindings for GNU parted 
- libffi-devel provides high level programming interface to various calling conventions